### PR TITLE
[DCFS-5] Create & Style the Customer Signup Page

### PR DESCRIPTION
**Pull request by Keicee32**
## Task: `Create & Style the Customer Signup Page`
- `Created the signup page and styled it using the sample format given`

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done 

- All images should be inside the `images` folder that is in your `html` folder
- `signup.html` should be inside `html-forms` folder.
- Also, kindly attach a screenshot of your work on the description part of your PR.
- Kindly fix codefactor issues on your PR

**Comment by Keicee32** [comment]
Screenshot (33)
ScreenShot

File: submissions/egekenzeKelechi/style.css [hunk 1]
```diff
@@ -0,0 +1,223 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```
Comment (adaugochi) [inline submissions/egekenzeKelechi/style.css:1]: your `style.css` file should be inside `css` folder

File: submissions/egekenzeKelechi/style.css [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+    button{
+        margin-bottom: 20px;
+        font-size: 18px;
+    }
+
+}
+
+/* 
+.container{
```
Comment (adaugochi) [inline submissions/egekenzeKelechi/style.css:160]: kindly fix codefactor issue on your PR

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/egekenzeKelechi/style.css [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+    display: flex;
```
Comment (adaugochi) [inline submissions/egekenzeKelechi/style.css:161]: Why are you commenting out this CSS code?

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Keicee32** [comment]
Screenshot (39)

Browser view of my work as requested

**Comment by ToluAkin** [comment]
@Keicee32 This is not the design for this task. I see you have designed the signup for zeplin. You need a zeplin account to see the design for the task.

**Pull request closed without merging**